docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -0,0 +1,176 @@
+----
+
+NOTE: You can also use xref:getting-started:get-started-binary.adoc#step-3-write-data-to-memory[alternative ways] to create a map and put data.
+
+=== Step 3. Start the Target Cluster
+
+The target cluster must be of version Hazelcast 5.3.2 or newer.
+
+Run the following Docker command to start a one-member cluster.
+
+[source,shell]
+----
+docker run -e HZ_NETWORK_PORT_PORT=5901 -p 127.0.0.1:5901:5901 
+           -e HZ_LICENSEKEY="<license>" <1>
+           -e HZ_CLUSTERNAME=target hazelcast/hazelcast-enterprise:{full-version}-slim <2>
+----
+<1> You can find the license key in the `enterprise-trial-license-key.txt` file included in the DMT download package.
+<2> If you want to use the full distribution, remove the `-slim` postfix. See xref:getting-started:editions.adoc#full-and-slim-distributions[Distributions] to learn about the differences between full and slim distributions.
+
+Alternatively, you can use the migration build which is provided in the DMT download package as the target cluster.
+Run the following command in the folder where you extracted the DMT package.
+
+```shell
+$ HZ_CLUSTERNAME=target HZ_NETWORK_PORT_PORT=5901 ./bin/hz start
+```
+
+The cluster tries to start on port `5901` if it's available. It chooses another port otherwise. You can check the port in the cluster logs shown on the terminal.
+
+The configuration file for the target cluster is `target.yaml`, which can be found in the folder where you extracted the DMT package.
+In case the cluster starts on a different port as you notice within the cluster logs, you must change the `address` field in `target.yaml` accordingly to match the port number.
+
+=== Step 4. Start the Migration Cluster
+
+Run the following command in the folder where you extracted the DMT package.
+
+[source,shell]
+----
+$ HZ_NETWORK_PORT_PORT=5702 HZ_CLUSTERNAME=migration ./bin/hz start
+----
+
+The cluster tries to start on port `5702` if it's available. It chooses another port otherwise. You can check the port in the cluster logs shown on the terminal.
+
+The configuration file for the migration cluster is `migration.yaml`, which can be found in the folder where you extracted the DMT package.
+In case the cluster starts on a different port as you notice within the cluster logs, you must change the `address` field in `migration.yaml` accordingly to match the port number.
+
+=== Step 5. Start the Migration using DMT
+
+Go to `bin` in the folder where you extracted the DMT package; there are DMT binaries in the format `dmt_[platform]_[arch]`. Note the binary suitable 
+for your machine; you need to know your operating system and your processor architecture. For `arm`, choose the suitable `arm64` binary, and for Intel, choose the suitable `amd64` binary.
+
+Run the following command in the folder where you extracted the DMT package, to start the migration.
+
+[source,shell]
+----
+$ ./bin/dmt_[platform]_[arch] --config migration.yaml start migration_config --yes
```

Comment:
We should explain the migration_config folder and what's in there as well. 

Basically, we have source and target folders. These folders are for files required to connect to source and target clusters by the data migration service running on the migration cluster. There has to be a hazelcast-client.yaml file in each folder. Later on, xml will be supported as well. If both xml and yaml exist, the same priority used by Hazelcast is used. 

Files in source and target folders other than hazelcast-client.yaml file are transferred as they are. For example, the users can put their certificate files next to hazelcast-client.yaml and they will be transferred and therefore migration cluster will use SSL/TLS to connect to source or target cluster. A note about certificates: The truststore and keystore paths in hazelcast-client.yaml file have to relative, not absolute. 

The data folder is for selecting which data structures the migration should migrate. There are two files in this folder:

imap_names.txt: IMap names to migrate, one at each line

replicated_map_names.txt: ReplicatedMap names to migrate, one at each line

Right now, it is not supported but we will support wildcards in IMap or Rep. Map names. It will work the same way as our current wildcard support

Reply:
We don't have to explain the folder structure in this tutorial. In general, we should talk about it somewhere